tests/testthat/test-get_pvalue.R

```diff
@@ -38,6 +38,24 @@ testthat::test_that("T1.1. No error when a `survfit` object is passed to the fun
 
   survfit_object <- visR::estimate_KM(adtte, strata = "TRTA")
   testthat::expect_error(visR::get_pvalue(survfit_object), NA)
+
+  # testing the p-value is correct when filtered data is piped to estimate_KM()
```

Comment:
this should be  a new test

Reply:
also do a test within estimate_KM for this

Reply:
upcate cox testing too

Reply:
Added this test to its own section, and added similar tests in `estimate_KM()` as well. The cox model hasn't been merged yet.